src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp

```diff
@@ -1263,6 +1864,30 @@ INSTANTIATE_TEST_SUITE_P(smoke_JIT_AVX512_DW_GroupConv, GroupConvolutionLayerCPU
 /* ============= JIT AVX5122 PLANAR Convolution (not supported with groups) ============= */
 /* ============================================= */
 
+/* ============= expect failed brgemm GroupConvolution  ============= */
+const std::vector<CPUSpecificParams> CPUParams_Failed_Brgemm_2D = {
+        conv_avx512_2D_nspc_brgconv,
+        conv_avx512_2D_nspc_brgconv_amx
+};
+const std::vector<CPUSpecificParams> CPUParams_Failed_Brgemm_1D_Small_Shape = {
+        conv_avx512_1D_nspc_brgconv_amx
+};
+const std::vector<groupConvLayerCPUTestParamsSet> BRGEMM_EXPECT_FAILED_GroupConvTestCases = generateSingleGroupConvCPUTestCases(
+        // channel <= 16
+        // https://github.com/openvinotoolkit/oneDNN/blob/6df930dab5ab0a7dfaea6100acd03b479e2fa0a8/src/cpu/x64/jit_brgemm_conv_utils.cpp#L1712
```

Comment:
I propose to delete those references to the code of particular commit, they will become obsolete soon.

Reply:
The problem is that after few months oneDNN could decide to refactor the piece of code we are referring to, and having this reference won't help much, because we will have to deal with an "updated" implementation anyway.
Leaving it to @dmitry-gorokhov whether we need to have a direct link to this piece of oneDNN code.